src/ds5/ds5-motion.h

```diff
@@ -200,11 +200,16 @@ namespace librealsense
         {
             imu_calib_table = *(ds::check_calib<ds::dm_v2_calibration_table>(raw_data));
 
-            // TODO - need to check mechanical drawing for extrinsic and orientation
             // Bosch BMI055
-            // L515 specific - BMI055 assembly transformation based on mechanical drawing (mm)
-            _def_extr = { { 1, 0, 0, 0, 1, 0, 0, 0, 1 },{ -0.01245f, 0.01642f, 0.02093f } };
-            _imu_2_depth_rot = { { 1,0,0 },{ 0,1,0 },{ 0,0,1 } };
+            // L515 specific - BMI055 assembly transformation based on mechanical drawing (meters)
```

Comment:
Doesn't the L515 use a BMI085 (not 055)?

Reply:
thanks for the review, corrected.

regarding your other question on the rotation matrix, I added additional comments in the code to describe the convention. the imu position and output is aligned to d435i output, so the same matrix should be used to follow the same output convention described in the comments.

the same changes already included as part of another PR (https://github.com/IntelRealSense/librealsense/pull/6524) and will be included in the next release, so this PR will be discarded as the changes are duplicated.

please let us know if you encounter any issue or question. thanks again for the feedback.